monitoring/prometheus/alerts/errors.yml

```
@@ -0,0 +1,21 @@
+# Process-wide error surface. Epic 13 T14.
+groups:
+  - name: demos-errors
+    rules:
+      - alert: UncaughtExceptionSpike
+        expr: increase(demos_uncaught_exception_total[5m]) > 10
+        for: 1m
+        labels:
+          severity: warning
+        annotations:
+          summary: "Spike of uncaught exceptions"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)